test/components/bbox_polygon.dart

```diff
@@ -0,0 +1,59 @@
+import 'package:test/test.dart';
+import 'package:turf/helpers.dart';
+import 'package:turf/src/bbox_polygon.dart';
+import 'package:turf/src/bbox.dart';
+
+main() {
+  test(
+    "bbox-polygon",
+    () {
+      var poly = bboxPolygon(Bbox(0, 0, 10, 10));
+      expect((poly.geometry is Polygon), equals(true));
+    },
+  );
+
+  test(
+    "bbox-polygon -- valid geojson",
+    () {
+      var poly = bboxPolygon(Bbox(0, 0, 10, 10));
+      var coordinates = poly.geometry!.coordinates;
+
+      expect((coordinates[0].length == 5), equals(true));
```

Comment:
you don't need the extra parentheses around the condition on the left side and also not the `equals(...)` surrounding the true on the right side